draft-ietf-dnsop-svcb-https.md

```diff
@@ -670,6 +664,15 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+To improve performance when using a non-conforming recursive resolver,
+clients SHOULD issue speculative A and AAAA queries based on a predicted value
+of TargetName.  In the absence of better information, clients SHOULD predict
+that TargetName will be the service name for the initial query and "." for
+subsequent queries (see {zone-performance}).
```

Comment:
This isn't something you recommend.  You just need to say that the reason you issue A/AAAA in parallel is that you are predicting that TargetName is present and that TargetName is ".", which is what {{zone-performance}} (two curlies!) recommends for those that provision records.

Reply:
{{zone-performance}} actually didn't recommend that.  I've moved this recommendation from here to there.